.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: go
 go:
-  #- 1.5  go test arguments are different in 1.5 :-(
-  - 1.6
```

Comment:
I was running 1.6 up until very recently... I think we are good on 1.6.. maybe remove it after we get reports of it breaking?